docs/resources/monitor_alert.md

```diff
@@ -0,0 +1,80 @@
+---
+page_title: "DigitalOcean: digital_monitor_alert"
+---
+
+# digitalocean_monitor
+
+Provides a [DigitalOcean Monitoring](https://docs.digitalocean.com/reference/api/api-reference/#tag/Monitoring) resource.
+Monitor alerts can be configured to alert about, e.g., disk or memory usage exceeding certain threshold, or traffic at certain
+limits. Notifications can be sent to either an email address or a Slack channel.
+
+### Basic Example
+
+```hcl
+# Create a new Web Droplet in the nyc2 region
+resource "digitalocean_droplet" "web" {
+  image  = "ubuntu-20-04-x64"
+  name   = "web-1"
+  region = "nyc2"
+  size   = "s-1vcpu-1gb"
+}
+
+resource "digitalocean_monitoring" "cpu_alert" {
+  alerts      {
+    email = ["benny@digitalocean.com"]
+    slack {
+      channel   = "Production Alerts"
+      url       = "https://hooks.slack.com/services/T1234567/AAAAAAAA/ZZZZZZ"
+  }
+  window      = "5m"
+  type        = "v1/insights/droplet/cpu"
+  compare     = "GreaterThan"
+  value       = 95
+  enabled     = true
+  entities    = [digitalocean_droplet.web.id]
+  description = "Alert about CPU usage"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `alerts` - (Required) How to send notifications about the alerts.
```

Comment:
If we stick with alerts being a list, then we should indicate here that only one alerts element is allowed.

Reply:
Yes. I've configured some alert policies inside the web panel. I only used email to get notifications, which is a single list (fine in this PR), but I'm not sure about multiple slack channels. So yes, we can clarify the exact phrasing here.